source/apiVolontaria/volunteer/serializers.py

```diff
@@ -403,6 +403,7 @@ class ParticipationBasicSerializer(serializers.ModelSerializer):
 
     # Explicitly declare the BooleanField to make it "required"
     standby = serializers.BooleanField()
+    send_ics = serializers.BooleanField(required=False, read_only=True)
```

Comment:
I think this field is `write_only` and not `read_only`

Reply:
It's working with read_only, write_only send this error : TypeError: 'send_ics' is an invalid keyword argument for this function

Reply:
It's really strange, did you use this kind of code ?

```python
extra_kwargs = {'password': {'write_only': True}}
```

Reply:
No, I didn't.